packages/server/graphql/queries/helpers/fetchAllIntegrations.ts

```diff
@@ -10,15 +13,16 @@ const fetchAllIntegrations = async (
   context: any,
   info: GraphQLResolveInfo
 ) => {
-  const results = await Promise.allSettled([
+  const results = (await Promise.allSettled([
     fetchAtlassianProjects(dataLoader, teamId, userId),
     fetchGitHubRepos(teamId, userId, dataLoader, context, info)
-  ])
-  const allIntegrations = results.flatMap((result) =>
-    result.status === 'fulfilled' ? result.value : []
-  )
+  ])) as PromiseSettledResult<Integration[]>[]
```

Comment:
If I remove this casting, results is type:
```
[PromiseSettledResult<JiraProject[]>, PromiseSettledResult<{
    id: string;
    service: string;
    nameWithOwner: string;
}[]>]
```
TypeScript complains on line 21 that `Type 'JiraProject[] | { id: string; service: string; nameWithOwner: string; }[]' is not assignable to type 'JiraProject | readonly JiraProject[]'.` I spent some time digging into a better solution without much luck, so I've opted for this for now.

Reply:
+1 we can just use `Promise.all` here because we're guaranteed neither will throw